control-plane/files/values/rancher_chart_values.tftpl

```diff
@@ -21,4 +21,4 @@ bootstrapPassword: ${rancher_password}
 %{ endif ~}
 extraEnv:
 - name: CATTLE_PROMETHEUS_METRICS
-  value: 'true'
\ No newline at end of file
+  value: '${cattle_prometheus_metrics}'
```

Comment:
Are we feeding these variables from somewhere? And by any chance, is it there a job that's executing these TF scripts?

Reply:
> Are we feeding these variables from somewhere? And by any chance, is it there a job that's executing these TF scripts?

This is coming from [control-plane/modules/install-common/helm_release.rancher](https://github.com/rancher/rancher2-scaling/blob/1c033d99d82a07386f5716ad71e323b9540637ce/control-plane/modules/install-common/main.tf#L75). Unfortunately there isn't a job/automated builder for this repo but there is a task in the backlog to work on getting that going.